frontend/src/components/HelpPopup/HelpPopup.test.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { render, fireEvent } from "@testing-library/react";
+// import userEvent from "@testing-library/user-event";
+
+import HelpPopup from "./HelpPopup";
+import userEvent from "@testing-library/user-event";
+
+jest.mock(
+  "./HelpPopup",
+  () =>
+    function () {
+      return <span>HelpPopup</span>;
+    },
+);
```

Comment:
We should only use this if we care about the rendering of HelpPopup, but doing this would prevent us from testing its functionalities as you are mocking the whole component here to only render HelpPopup. 

So I think  you should remove this